media-libs/sdl-net/sdl-net-1.2.8-r2.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit multilib-minimal toolchain-funcs
+
+MY_P="${P/sdl-/SDL_}"
+DESCRIPTION="Simple Direct Media Layer Network Support Library"
```

Comment:
Prefer a clear cut for the description/homepage block, so I usually have a blank line between these.